app/models/product_import/product_importer.rb

```diff
@@ -57,6 +57,13 @@ def item_count
       @sheet ? @sheet.last_row - 1 : 0
     end
 
+    def product_field_errors?
+      @entries.each do |entry|
+        return true if entry.errors.messages.values.include? [I18n.t('admin.product_import.model.not_updatable')]
+      end
+      false
+    end
```

Comment:
Using [any?](https://ruby-doc.org/core-2.2.0/Enumerable.html#method-i-any-3F) would be a bit more readable IMO.

Reply:
I meant using any? with `@entries`. If I understood correctly, we're checking if any of the entries has an error regardless of the number of errored ones, right?